packages/design-system/src/components/RichRadioButton/RichRadioButton.module.scss

```diff
@@ -46,7 +47,7 @@
 		cursor: pointer;
 		height: 100%;
 		left: 0;
-		margin: 0;
+		margin: 0 !important;
```

Comment:
You shouldn't need the !important. Is there any reason why you added it? 😅

Reply:
Yes, there is a global style via `variables.css` (so it seems) for every input type radio, applying a `margin-top: 4px`.